### PR TITLE
Splits in EURW with hyphens

### PR DESCRIPTION
**Pull request by ClipperWings**

### CODE REVIEW

**Comment by pierr3** [comment]
Confirming that LFFF is not relevant for VATSpy in this context

File: VATSpy.dat [hunk 1]
```diff
@@ -18420,7 +18420,8 @@ EURN|Eurocontrol North|EKDK,EETT,EFIN,EVRR,EYVL,ENBD,ENOS,ENSV,ESOS
 EURM|Maastricht Eurocontrol|LOVV,EBBU,EDGG,EDMM,EDWW,LSAS,EHAA
 EURM-E|Maastricht Eurocontrol|LOVV,EDMM,EDWW-B,LKAA
 EURM-W|Maastricht Eurocontrol|EBBU,EDGG,EDWW-A,LSAS,EHAA
-EURW|Eurocontrol West|LFBB,LFEE,LFMM,LFRR,LFFF,LPPC,LECM,LECB,LECS
+EURW-N|Eurocontrol West|LFBB,LFEE,LFMM,LFRR,LFFF
```
[the remark this answers is not in the document]
Comment (Adrian2k) [reply VATSpy.dat:18423]: AIRAC date in two days, please get this resolved as soon as possible.

[the remark this answers is not in the document]
Comment (pierr3) [reply VATSpy.dat:18423]: I confirm removing LFFF is correct

[the remark this answers is not in the document]
Comment (ClipperWings) [reply VATSpy.dat:18423]: Oups ! Can you guys remove it or should it be us as we're both confirming LFFF should be removed ?

**Review by Adrian2k — APPROVED** [review]

**Review by NelisV — APPROVED** [review]

**Pull request merged**